apps/zbugs/shared/queries.ts

```diff
@@ -42,39 +42,63 @@ const issueRowSort = z.object({
 
 type IssueRowSort = z.infer<typeof issueRowSort>;
 
+const defineAuthQuery = defineQueryWithContextType<AuthData | undefined>();
+
 export const queries = {
-  allLabels: syncedQuery('allLabels', z.tuple([]), () => builder.label),
+  allLabels: defineQuery(
+    'allLabels',
+    {
+      validator: z.undefined(),
+    },
+    () => builder.label,
+  ),
 
-  allUsers: syncedQuery('allUsers', z.tuple([]), () => builder.user),
+  allUsers: defineQuery(
+    'allUsers',
+    {
+      validator: z.undefined(),
+    },
+    () => builder.user,
+  ),
 
-  allProjects: syncedQuery('allProjects', z.tuple([]), () => builder.project),
+  allProjects: defineQuery(
+    'allProjects',
+    {
```

Comment:
Now that I see this it does seem kind of silly to require passing `{validator` when there is just one field right now.